frontend/src/components/core/StatusWidget/StatusWidget.tsx

```diff
@@ -450,4 +455,5 @@ class StatusWidget extends PureComponent<StatusWidgetProps, State> {
   }
 }
 
-export default StatusWidget
+// @ts-ignore
```

Comment:
I spent too long trying to get this to work and gave up but if you have ideas to make it work, that'll be great!

![image](https://user-images.githubusercontent.com/24946400/107096373-a3954e00-67d8-11eb-91d7-12f3056d748d.png)